gax-grpc/src/main/java/com/google/api/gax/grpc/AttemptCallable.java

```diff
@@ -33,26 +33,28 @@
 import com.google.api.core.ApiFutures;
 import com.google.api.gax.retrying.NonCancellableFuture;
 import com.google.api.gax.retrying.RetryingFuture;
+import com.google.api.gax.rpc.UnaryCallable;
 import io.grpc.CallOptions;
 import java.util.concurrent.Callable;
 import java.util.concurrent.TimeUnit;
 import org.threeten.bp.Duration;
 
 /**
- * A callable representing a retriable grpc call. This class is used from {@link RetryingCallable}.
+ * A callable representing a retriable grpc call. This class is used from {@link
+ * GrpcRetryingCallable}.
  *
  * @param <RequestT> request type
  * @param <ResponseT> response type
  */
 class AttemptCallable<RequestT, ResponseT> implements Callable<ResponseT> {
```

Comment:
I would prefer to rename this to GrpcAttemptCallable to be consistent with other grpc callable classes.